lib/recurly/requests/subscription_change_create.rb

```diff
@@ -3,8 +3,8 @@ module Requests
     class SubscriptionChangeCreate < Request
 
       # @!attribute add_ons
-      #   @return [Array[SubscriptionAddOnCreate]] If you set this value you include all the add-ons and their quantities and amounts. The values you include will replace the previous values entirely.
-      define_attribute :add_ons, Array, {:item_type => :SubscriptionAddOnCreate}
+      #   @return [Array[String]] If you set this value you include all the add-ons and their quantities and amounts. The values you include will replace the previous values entirely.
+      define_attribute :add_ons, Array, {:item_type => String}
```

Comment:
Wondering why this is treating the AddOns as an array of strings.

Reply:
It's possible that I hand-corrected this problem before. I'm going to take a look at it when I get some time today.